python/lsst/cp/pipe/cpElectrostaticBF.py

```diff
@@ -0,0 +1,498 @@
+# This file is part of cp_pipe.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Computation of electrostatic solution of brighter-fatter effect impact
+on pixel distortions"""
+
+__all__ = ['ElectrostaticBrighterFatterSolveTask',
+           'ElectrostaticBrighterFatterSolveConfig']
+
+import numpy as np
+
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+import lsst.pipe.base.connectionTypes as cT
+
+from .utils import (
+    extractCalibDate,
+    ElectrostaticFit,
+)
+from .cpLinearitySolve import ptcLookup
+from lsst.ip.isr.isrFunctions import symmetrize
+from lsst.ip.isr import ElectrostaticBrighterFatterDistortionMatrix
+from lmfit import Parameters, report_fit
+
+
+class ElectrostaticBrighterFatterSolveConnections(pipeBase.PipelineTaskConnections,
+                                                  dimensions=("instrument", "detector")):
+    dummy = cT.Input(
+        name="raw",
+        doc="Dummy exposure.",
+        storageClass='Exposure',
+        dimensions=("instrument", "exposure", "detector"),
+        multiple=True,
+        deferLoad=True,
+    )
+    camera = cT.PrerequisiteInput(
+        name="camera",
+        doc="Camera associated with this data.",
+        storageClass="Camera",
+        dimensions=("instrument", ),
+        isCalibration=True,
+    )
+    inputPtc = cT.PrerequisiteInput(
+        name="ptc",
+        doc="Photon transfer curve dataset.",
+        storageClass="PhotonTransferCurveDataset",
+        dimensions=("instrument", "detector"),
+        isCalibration=True,
+        lookupFunction=ptcLookup,
+    )
+    inputBfPtc = cT.Input(
+        name="bfPtc",
+        doc="Input BF PTC dataset.",
+        storageClass="PhotonTransferCurveDataset",
+        dimensions=("instrument", "detector"),
+        isCalibration=True,
+    )
+
+    output = cT.Output(
+        name="electroBfDistortionMatrix",
+        doc="Output measured brighter-fatter electrostatic model.",
+        storageClass="ElectrostaticBrighterFatterDistortionMatrix",
```

Comment:
This is incorrect.  It should be `IsrCalib` *everywhere* in the storage class (inputs/outputs/isr/etc)